types/dockerode/index.d.ts

```diff
@@ -810,6 +810,18 @@ declare namespace Dockerode {
     BindOptions ?: {
       Propagation: MountPropagation;
     };
+    VolumeOptions ?: {
+      NoCopy: boolean;
+      Labels: {};
+      DriverConfig: {
+          Name: string;
+          Options: {};
```

Comment:
This would probably be better as `{ [name: string]: string }` based on https://docs.docker.com/engine/api/v1.39/#operation/ContainerCreate